test-harness/index.ts

```diff
@@ -58,10 +58,13 @@ describe('harness', () => {
 
           try {
             const isValid = validate(expected, output).valid;
-            if (!!isValid) expect(validate(expected, output).valid).toBeTruthy();
-            else {
-              expect(output).toMatchObject(expected);
+
+            if (!!isValid) {
+              expect(validate(expected, output).valid).toBeTruthy();
```

Comment:
```suggestion
              expect(isValid).toBeTruthy();
```

Reply:
This statement says 
`if (true) then expect(true).toBeTrue()` :) 
Anyway knows whats happening? ;) @XVincentX @lag-of-death

Reply:
This will make the test fail on purpose — because `isValid` in this condition, is going to be  always `false`

Reply:
`false`? it will be `true`

```
if (!!isValid) {
```
will pass if isValid is truthy.